src/test/java/integration/CounterpartyControllerIT.java

```diff
@@ -214,14 +217,14 @@ public void updateCounterparty() throws Exception {
         then().
                 contentType(APPLICATION_JSON_VALUE).
                 statusCode(SC_OK);
-        long timeFinished = currentTimeMillis();
+        LocalDateTime timeFinished = now();
 
         // check updated data
         Counterparty updatedCounterparty = counterpartyService.getEntityByUuid(counterpartyUuid, user);
         CounterpartyDto counterpartyDto = counterpartyMapper.toDto(updatedCounterparty);
-        long timeModified = updatedCounterparty.getLastModified().getTime();
+        LocalDateTime timeModified = updatedCounterparty.getLastModified();
 
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).